### PR TITLE
Update Restart-VM.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about wait and timeout parameters.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2662

File: docset/winserver2019-ps/hyper-v/Restart-VM.md [hunk 1]
```diff
@@ -218,6 +218,9 @@ Accept wildcard characters: False
 
 ### -Timeout
 
+Specifies the duration of the wait, in seconds. When the timeout elapses, Restart-VM returns to the command prompt, even if the VM isn't  restarted.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/hyper-v/Restart-VM.md:221]:
```suggestion
Specifies the duration of the wait, in seconds. When the timeout elapses, **Restart-VM** returns to the command prompt, even if the virtual machine isn't restarted.
```

File: docset/winserver2019-ps/hyper-v/Restart-VM.md [hunk 2]
```diff
@@ -218,6 +218,9 @@ Accept wildcard characters: False
...
+
+The Timeout parameter is only valid with the Wait parameter. Timeout overrides the Wait parameter's indefinite waiting period.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/hyper-v/Restart-VM.md:223]:
```suggestion
This parameter is only valid with the `-Wait` parameter. The `-Timeout` parameter overrides the `-Wait` parameter's indefinite waiting period.
```

File: docset/winserver2019-ps/hyper-v/Restart-VM.md [hunk 3]
```diff
@@ -248,6 +251,9 @@ Accept wildcard characters: False
 
 ### -Wait
 
+Restart-VM suppresses the PowerShell prompt and blocks the pipeline until the VM has restarted. You can use this parameter in a script to restart VM and then continue to process when the restart is finished.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/hyper-v/Restart-VM.md:254]:
```suggestion
**Restart-VM** suppresses the PowerShell prompt and blocks the pipeline until the virtual machine has restarted. You can use this parameter in a script to restart the virtual machine and then continue to process when the restart is finished.
```

File: docset/winserver2019-ps/hyper-v/Restart-VM.md [hunk 4]
```diff
@@ -248,6 +251,9 @@ Accept wildcard characters: False
...
+
+The Wait parameter waits indefinitely for the VM to restart. You can use Timeout to adjust the timing and the For and Delay parameters to wait for particular services to become available on the restarted VM.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/hyper-v/Restart-VM.md:256]:
```suggestion
The `-Wait` parameter waits indefinitely for the virtual machine to restart. You can use the `-Timeout` parameter to adjust the timing and the `-For` and `-Delay` parameters to wait for particular services to become available on the restarted virtual machine.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - these are only suggestions.

**Comment by JohanFreelancer9** [comment]
@msbemba, you have only committed one suggestion. Kindly commit my other suggestions as well. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by get-itips — COMMENTED** [review]
Can this information be added to other OS's versions of the article?

**Review by dariomws — APPROVED** [review]

**Comment by msbemba** [comment]
@get-itips It has been added to the other two versions where the parameter is available .

**Pull request merged**